packages/wallet/backend/src/grant/service.ts

```diff
@@ -80,6 +80,32 @@ export class GrantService implements IGrantService {
       args.filter = { identifier: { in: identifiers } }
     }
 
-    return await this.rafikiAuthService.listGrantsWithPagination(args)
+    const grants = await this.rafikiAuthService.listGrantsWithPagination(args)
+    this.parseIntervals(grants.grants.edges.map((it) => it.node))
+    return grants
+  }
+
+  private parseIntervals(grants: Grant[]) {
+    for (const grant of grants)
+      for (const accessElement of grant.access)
+        if (accessElement.limits?.interval) {
+          const time = accessElement.limits?.interval.split('/')
```

Comment:
why the number of repetition is ignored? isn't this piece important to show to the user